pom.xml

```diff
@@ -91,17 +91,17 @@
       <dependency>
         <groupId>com.google.dagger</groupId>
         <artifactId>dagger</artifactId>
-        <version>2.0</version>
+        <version>2.0.2</version>
```

Comment:
I'd rather keep it at `2.0` to not unexpectingly and silently upgrading someone to 2.0.2.